app/serializers/event_stream_serializers/classification_serializer.rb

```diff
@@ -12,6 +12,7 @@ def self.serialize(classification, options = {})
     belongs_to :user,             serializer: EventStreamSerializers::UserSerializer
     belongs_to :workflow,         serializer: EventStreamSerializers::WorkflowSerializer
     has_many   :subjects,         serializer: EventStreamSerializers::SubjectSerializer
+    type :classifications
```

Comment:
i think this should go above the relation links for readability, perhaps on line 10/11?